api-controllers/ServiceFabrikApiController.js

```diff
@@ -414,7 +415,13 @@ class ServiceFabrikApiController extends FabrikBaseController {
         } else if (backupGuid) {
           return this.validateUuid(backupGuid, 'Backup GUID');
         } else if (timeStamp) {
-          return this.validateDateString(timeStamp);
```

Comment:
I guess same comment as precedence of backup guid and timestamp applies here.

Reply:
done